.buildkite/pipeline.yml

```diff
@@ -120,21 +107,17 @@ steps:
     agents:
       queue: android
 
-  - label: "Build JS Bundles"
+  - label: Build JS Bundles
     depends_on:
       - lint
       - android-unit-tests
       - ios-unit-tests
-    key: "js-bundles"
+    key: js-bundles
     plugins:
-      - *gb-mobile-docker-container
+      - *ci_toolkit_plugin # unused?
```

Comment:
@mokagio If we are unsure if it's used, we could try to remove it in a separate PR to determine it. WDYT?